### ISSUE TITLE
How can I use hog with headers?

### ISSUE THREAD

**Issue by raccoonyy** [lqez/hog#5]
I tested like this.

```
$ hog -H test_header_key=test_header_value https://test.com/api/
```

But it failed.

```
usage: hog [-h] [-c CONCURRENCY] [-n REQUESTS] [-l LIMIT] [-t TIMEOUT]
           [-p [PARAMS [PARAMS ...]]] [-f PARAMFILE]
           [-H [HEADERS [HEADERS ...]]] [-F HEADERFILE] [-m {GET,POST}]
           url
hog: error: the following arguments are required: url
```

What did I wrong?

**Comment by lqez** [comment]
Thank you for reporting. I’ll check it out.

**Comment by lqez** [comment]
@raccoonyy basically, you can use it as `hog URL -H key=value` but the help message could be confused. And I found others who stuck in this problem. :(  https://stackoverflow.com/questions/13346540/argparse-optional-argument-before-positional-argument

**Comment by lqez** [comment]
@raccoonyy could you review https://github.com/lqez/hog/pull/6 ? I replaced `argparse` with `click`. With `click`, we do not have to get in trouble with positional argument issues.

**Comment by lqez** [comment]
I knew that there are cons and pros between `click` and `argparse`. But in this case, `argparse` displays its usage in an ambigious way. It shows like below but cannot accept `hog -H HEADERS url` format. As @raccoonyy's case, this causes unintended confusions.

```shell
usage: hog [-h] [-c CONCURRENCY] [-n REQUESTS] [-l LIMIT] [-t TIMEOUT]
           [-p [PARAMS [PARAMS ...]]] [-f PARAMFILE]
           [-H [HEADERS [HEADERS ...]]] [-F HEADERFILE] [-m {GET,POST}]
           url
```

So I changed it with `click`. I left this message for future developers.

**Comment by lqez** [comment]
Closed by #6
